src/Robo/Commands/Setup/ConfigCommand.php

```diff
@@ -39,8 +39,24 @@ public function import() {
       // If using core-only or config-split strategies, first check to see if
       // required config is exported.
       if (in_array($strategy, ['core-only', 'config-split'])) {
-        $core_config_file = $this->getConfigValue('docroot') . '/' . $this->getConfigValue("cm.core.dirs.$cm_core_key.path") . '/core.extension.yml';
 
+        // We don't want configs to be shared between multisites.
+        if ($this->getConfigValue('cm.share-configs') == FALSE) {
+          $drush_uri = $this->getConfigValue('drush.uri');
+
```

Comment:
@alex-moreno Are we sure 'drush.uri' is always set? If it's not shouldn't we fall back to 'default'? Also, is drush.uri the best variable to use here?

Reply:
niiiiice, good point. If we fall back to default/ you could have some sites using their isolated config, but some using the default. Love it.

Will commit the change in a second